Angular2-Starter-Project.md

```diff
@@ -0,0 +1,229 @@
+# Angular 2 Starter Project
+
+This tutorial will build an extremely simple seed project with Angular2. We will begin with the most basic application possible and then add on a bit more functionality.
+
+### Overview 
+
+Angular 2 applications are built by creating HTML templates that contain angular specific markup. Classes are then created to manage the templates and everything is wrapped into a module, which you piece together to create your application. These modules are referred to as **components** and are written in TypeScript.
+
+Angular interprets these modules and uses them to present your application in the browser.
+
+### Starter Application
+
+Let's get started with a minimal application. We will begin with an application that contains a single module whose only job is to display some text.
+
+We will begin by creating a new folder called "app".
+```
+mkdir app
+```
+As mentioned above, our application will begin with a single module, or **component**, that will display some text to the screen. Let's learn how to create a component using TypeScript. 
```

Comment:
**Add the following...**

We can use both pure JavaScript or TypeScript to make angular 2 app. Using TypeScript, it is easier and programmer friendly :wink: 
*Let's learn how to create a component using TypeScript.*